content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst

```diff
@@ -1,214 +1,99 @@
+  owns the Odoo database.
+- Two subcontracting routes become available in the *Inventory* app, and can be assigned to specific
+  products, on the *Inventory* tab of their product pages:
 
-.. image:: subcontracting/sbc_7.png
-    :align: center
-    :alt:
+  - *Resupply Subcontractor on Order*
+  - *Dropship Subcontractor on Order*
 
-Note that the buy route is not selected in this case, as the dropship route is
-a buy route already.
+Subcontracting workflows
+========================
 
-Finally, if you want to track the stock of these raw materials at your subcontracting
-location(s), then you must activate *Multi-locations* in :menuselection:`Inventory -->
-Configuration --> Settings --> Storage locations`.
+In Odoo, there are three subcontracting workflows, the main difference between them being *how the
+subcontractor obtains the necessary components*:
 
-From the location form, you are then able to access the Current Stock.
+- In the *basic* subcontracting workflow, the subcontractor is fully responsible for obtaining the
+  components. This workflow is outlined in the :doc:`subcontracting_basic` documentation.
+- In the *Resupply Subcontractor on Order* workflow, the contracting company sends the components
+  from their warehouse to the subcontractor. This workflow is outlined in the
+  :doc:`subcontracting_resupply` documentation.
+- In the *Dropship Subcontractor on Order* workflow, the contracting company purchases the
+  components from a vendor, and has them delivered directly to the subcontractor. This workflow is
+  outlined in the :doc:`subcontracting_dropship` documentation.
 
-.. image:: subcontracting/sbc_8.png
-    :align: center
-    :alt:
+In addition to how a subcontractor obtains components, it is also necessary to consider why a
+product is being subcontracted, as well as what happens to products once they are manufactured by
+the subcontractor.
 
+In terms of why a product is being subcontracted, the two main reasons are to fulfill a customer
+order, or to replenish the quantity of stock on-hand.
 
+In terms of what happens to products once they are manufactured, they can either be shipped to the
+contracting company, or dropshipped directly to an end customer.
 
-Manual Replenishment
---------------------
+Each of the three subcontracting workflows described above can be configured to facilitate any of
+these possibilities, and the methods for doing so are outlined in their respective documentation.
 
-You can also choose to replenish your subcontractors manually.
+Subcontracted product valuation
+===============================
 
-If you want to send components to your subcontractor at your own convenience,
-select the 'Resupply Subcontractor' Operation Type from the *Inventory* Module,
-and create a picking, specifying to which subcontractor you are delivering to.
+The valuation of a subcontracted product depends upon a few different variables:
```

Comment:
Depends on how you like to present information, but in math textbooks, we are typically presented the formula first, then the explanation of the variables. For example:

> The valuation of a subcontracted product is calculated using the following formula:
P = C + M + S + D + x

> **C** refers to the cost of the required components, if provided by the contracting company.

if you intend to keep the info presented in this format, I personally think that the variable definition is a little vague: "from here on referred
  to a `C`"
  
  How about: "represented by `C` in the formula"?